sphinx/ext/rose_domain.py

```diff
@@ -0,0 +1,721 @@
+"""The rose domain is for documenting rose configurations and built-in
```

Comment:
Please add standard header to Python files.

Reply:
Done.